homeassistant/components/overkiz/__init__.py

```diff
@@ -57,7 +58,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     try:
         await client.login()
-        setup = await client.get_setup()
+
+        tasks = [
+            client.get_setup(),
+            client.get_scenarios(),
+        ]
+        setup, scenarios = await asyncio.gather(*tasks)
```

Comment:
```suggestion
        setup, scenarios = await asyncio.gather(*[
            client.get_setup(),
            client.get_scenarios(),
        ])
```

Since you don't care about the tasks list after its run, you could do this

Reply:
f832173